src/jit/lower.cpp

```diff
@@ -3109,6 +3116,8 @@ void Lowering::InsertPInvokeMethodProlog()
     GenTree* lastStmt = stmt;
     DISPTREE(lastStmt);
 
+#ifndef _TARGET_X86_ // For x86, this step is done at the call site.
```

Comment:
Please also mention the reason for doing this step at call site (i.e. due to floating stack pointer)